test/extended/util/db/mongodb.go

```diff
@@ -0,0 +1,60 @@
+package db
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/origin/test/extended/util"
+)
+
+// MongoDB is a MongoDB helper for executing commands.
+type MongoDB struct {
+	podName       string
+	masterPodName string
+}
+
+// NewMongoDB queries OpenShift for a pod with given name, saving environment
+// variables like username and password for easier use.
```

Comment:
I think this godoc is wrong. This function just return a struct (it "creates a new util.Database instance"), it doesn't "query OpenShift"...